src/prisma/generator/templates/engine/query.py.jinja

```diff
@@ -46,16 +46,20 @@ class QueryEngine(HTTPEngine):
     def __del__(self) -> None:
         self.stop()
 
-    def close(self) -> None:
+    def close(self, timeout: float = 1.0) -> None:
```

Comment:
I'm not sure if this is the right timeout but I also don't have a better suggestion....

I think for now it might be best to make this optional and default to `None` and support passing this through from `db.disconnect()` so you can set the default to `1` if that makes sense in your application.

Reply:
Makes total sense. I will also revert the `Popen[str]` type definition since it is still failing... I will try setting it to `Popen[str]` everywhere...